content/docs/guides/vercel.md

```diff
@@ -75,12 +89,6 @@ Neon’s branching feature addresses all of these challenges. A branch is a copy
 
 When you push a branch to the GitHub repository associated with your Vercel project, triggering a preview deployment, the integration automatically creates a database branch in Neon and connects it to your preview deployment by setting the required Vercel preview environment variables. The newly created Neon branch will have same name as the Git branch containing the code changes.
 
-<Admonition type="important">
```

Comment:
why remove this part?

Reply:
My understanding is that this statement is not quite correct: "For the preview deployment integration to work, the database connection settings in your application must correspond to the Vercel production environment variable settings configured by your Neon integration." If I understand correctly, it's just the environment variable settings in Vercel that dictate whether the integration works. I'll follow up with another PR to address this.